### DISCUSSION TITLE
v8.0-(1403-3-18)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#48] [Announcements]
[mahsa_net telegram](https://t.me/mahsa_net)
[mahsanet twitter](https://twitter.com/mahsanet)

نوع پردازنده گوشی خود را از پایین صفحه تنظیمات تلگرام ببینید
یا اینکه نسخه جنریک نصب کنید سازگار با تمام پردازنده ها

optimize xray-core for iran
unblock fragment tlshello
unblock warp and wireguard
revive youtube config
lots of bug fixed

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v8.0-(1403-3-18)'>v8.0-(1403-3-18)</a>.</em>

**Comment by Sleepy1195** [comment]
سلام من نسخه ۸ نصب کردم هیچ وایرگادی وصل نشد.دوباره نسخه ۷ رو نصب کردم ولی با ۷ وصل شدن....انگار یه باگ داره

**Reply by GFW-knocker** [reply]
روی چه نتی تست کردید؟
وارپ اختلال سنگینی داره که هر منطقه یه داستانه. ممکنه با تعویض ایپی یا تغییر پورت به 928 یا 955 مشکل حل بشه.
ممکنه روش اتصال نسخه 7 رو نت شما بصورت موردی بهتر باشه

**Reply by Sleepy1195** [reply]
روی نت همراه اول تست کردم
همون کانفیگی که روی نسخه ۷ جواب میده ولی روی نسخه ۸ جواب نمیده
کلی کانفیگ تست کردم روی نسخه ۸ ولی وصل نشد با وایرگاد
ولی روی نسخه ۷ همه وصل شدن

**Reply by kekelove1688** [reply]
大佬们，有windows版本开发吗

**Reply by GFW-knocker** [reply]
> 大佬们，有windows版本开发吗

windows version planned but take few month to release.
until then you can use mahsa xray-core if you need fragment, wireguard or fakehost
https://github.com/GFW-knocker/Xray-core

**Comment by Sleepy1195** [comment]
روی نت همراه اول تست کردم
همون کانفیگی که روی نسخه ۷ جواب میده ولی روی نسخه ۸ جواب نمیده
کلی کانفیگ تست کردم روی نسخه ۸ ولی وصل نشد با وایرگاد
ولی روی نسخه ۷ همه وصل شدن

**Reply by GFW-knocker** [reply]
ممنون بابت اطلاع. بررسی میشه.

**Reply by Sleepy1195** [reply]
ممنونم از برنامه عالیتون.اگر این مشکل روی نسخه ۸ حل بشه که بتونیم همه جامپ کنیم روی نسخه جدید 
تشکر

**Comment by bni021** [comment]
مرسی بابت زحماتتون ممنونیم
